### ISSUE TITLE
Align education pages with enrollment verification tool landing page

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#57073]
## Issue Description

Now that the enrollment verification tool is live, we need to look at other education pages to make sure content aligns with the new [Verify school enrollment for GI Bill benefits](https://www.va.gov/education/verify-school-enrollment/) page. We also need to add cross links throughout.

The pages to review:

- [Post-9/11 GI Bill](https://www.va.gov/education/about-gi-bill-benefits/post-9-11/)
- [MGIB-AD](https://www.va.gov/education/about-gi-bill-benefits/montgomery-active-duty/)
- [MGIB-SR](https://www.va.gov/education/about-gi-bill-benefits/montgomery-selected-reserve/)
- [VET TEC](https://www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/)
- [STEM Scholarship](https://www.va.gov/education/other-va-education-benefits/stem-scholarship/)

## Tasks
- [x] Review pages listed above and align content (Katherine)
- [x] Add crosslinks throughout (Katherine)
- [x] OCTO review (Beth)
- [x] Share pages with stakeholders (Katherine)
- [x] Copyedit (Megan)
- [x] Publish (Megan)

## Acceptance Criteria
- [ ] Pages are live with updates

**Comment by katherine-fung** [comment]
Veterans technology education courses 
http://preview-prod.vfs.va.gov/preview?nodeId=930

Edith Nourse Rogers STEM Scholarship
http://preview-prod.vfs.va.gov/preview?nodeId=6860

**Comment by katherine-fung** [comment]
Hi @bethpottsVADEPO, 

I've reviewed the program pages. 

The Post-9/11 GI Bill already has a cross link to the "Verify school enrollment for GI Bill benefits" page. 
MGIB-AD and MGIB-SR pages also have cross links to the WAVE tool (the link text is: "Verify your enrollment online") 

For the remaining STEM and VET TEC pages, I added a section that says: 

> How do I get my monthly housing allowance? 
> 
> We'll send your monthly housing allowance (MHA) directly to you every month. 
> 
> You’ll need to verify your enrollment every month to get payments. 
> 
> [LINK] Learn how to verify your enrollment for the STEM scholarship
> 

Here are the pages to review: 
VET TEC: 
http://preview-prod.vfs.va.gov/preview?nodeId=930 

STEM scholarship: 
http://preview-prod.vfs.va.gov/preview?nodeId=6860

On the "Verify school enrollment for GI Bill benefits" page, I also added a few details about what you need to include in your email when you verify VET TEC enrollment. We'll need to review and publish those changes on that page. 

Here's the inline comparison: https://prod.cms.va.gov/node/51828/revisions/view/752943/785817/visual_inline 

Thank you!

**Comment by bethpottsVADEPO** [comment]
Thanks, @katherine-fung! 
1/ Great new Q&A. I think I'd just remove "every month" from the first sentence as I think this is clear enough since the benefit is "monthly housing allowance" and the second sentence has "every month." 

2/ I'd like to move a couple things...

**Veterans technology education courses**
http://preview-prod.vfs.va.gov/preview?nodeId=930
For this page, I think it makes more sense for the new "How do I get my monthly housing allowance?" Q&A to follow "What benefits can I get through this program?" Would you move that Q&A up?

**Edith Nourse Rogers STEM Scholarship**
http://preview-prod.vfs.va.gov/preview?nodeId=6860
This page is currently organized atypically: Let's move "What benefits can I get with this scholarship?" up above "How do I apply" and then add your new Q&A following "What benefits can I get..." (similar order of events to VET TEC).

**Verify school enrollment for GI Bill benefits**
http://preview-prod.vfs.va.gov/preview?nodeId=51828
Looks good! I made a minor tweak to the set-up line to the 2 bullets for what to include in your email.

**Comment by katherine-fung** [comment]
Thank you @bethpottsVADEPO. 

As we discussed yesterday, I also added a crosslink for the new "Verify your school enrollment for DEA benefits" page on the DEA program page. 

I've scheduled an email to stakeholders on Monday morning letting them know about the updates. I told them we'd be publishing very soon. I'll update this ticket if they have any concerns. Otherwise, these pages are ready for copy editing when Megan is back next week. 

These pages to copy edit are: 
1. Verify school enrollment for GI Bill benefits - 
I added instructions for how to verify your enrollment by email for VET TEC.
http://preview-prod.vfs.va.gov/preview?nodeId=51828

2. Veterans technology education courses - 
I added a  "How do I get my monthly housing allowance?" Q&A.
http://preview-prod.vfs.va.gov/preview?nodeId=930

3. Edith Nourse Rogers STEM Scholarship - 
I added a "How do I get my monthly housing allowance?" Q&A. 
http://preview-prod.vfs.va.gov/preview?nodeId=6860

4. Survivors and Dependents' Educational Assistance - 
I added a "How do I get my monthly payment?" Q&A here. 
http://preview-prod.vfs.va.gov/preview?nodeId=950

**Comment by katherine-fung** [comment]
Hi @megzehn, 

These 4 pages are ready to copy edit and publish. The stakeholders have seen the changes. Thanks! 


1. Verify school enrollment for GI Bill benefits -
I added instructions for how to verify your enrollment by email for VET TEC.
http://preview-prod.vfs.va.gov/preview?nodeId=51828

2. Veterans technology education courses -
I added a "How do I get my monthly housing allowance?" Q&A.
http://preview-prod.vfs.va.gov/preview?nodeId=930

3. Edith Nourse Rogers STEM Scholarship -
I added a "How do I get my monthly housing allowance?" Q&A.
http://preview-prod.vfs.va.gov/preview?nodeId=6860

4. Survivors and Dependents' Educational Assistance -
I added a "How do I get my monthly payment?" Q&A here.
http://preview-prod.vfs.va.gov/preview?nodeId=950
